daemon/src/lib.rs

```diff
@@ -202,11 +201,7 @@ where
 
         let (endpoint_addr, endpoint_context) = Context::new(None);
 
-        ping::Actor::new(endpoint_addr.clone(), PING_INTERVAL)
-            .create(None)
-            .spawn(&mut tasks);
         let pong_address = pong::Actor::default().create(None).spawn(&mut tasks);
-
```

Comment:
I feel like this should be a separate PR, as you suspect that it's a contentious change - let's chat about it separately.
the main reason is: it's hard to observe the possible congestions you're mitigating in this PR  if you disable that side of the protocol. 

all in all, I'm happy to explore alternatives, but I don't think they'll be easier - the one outlined in the commit message has the same drawback that the old heartbeat (intimate knowledge how pings on the other party are set up, ie. how often do they ping). Pinging from the taker overcomes this. If anything, I'd consider removing the heartbeat from the *maker* side - we don't really do anything with the latency measurements.

Also, we're not using this information *yet*, because we still rely on heartbeat - hopefully not for a long time.

Reply:
> I feel like this should be a separate PR, as you suspect that it's a contentious change - let's chat about it separately.
the main reason is: it's hard to observe the possible congestions you're mitigating in this PR if you disable that side of the protocol.

I am happy to move it. I was just lazy last night.

I just made a lot of observations using [the test](https://github.com/itchysats/itchysats/commit/5882fd5851d3a36b886f651edd4a17b58154c44a) and by far the biggest difference with the `1-to-m` actor setup was to not have the `m`s pinging too.

> all in all, I'm happy to explore alternatives, but I don't think they'll be easier - the one outlined in the commit message has the same drawback that the old heartbeat (intimate knowledge how pings on the other party are set up, ie. how often do they ping). Pinging from the taker overcomes this. If anything, I'd consider removing the heartbeat from the maker side - we don't really do anything with the latency measurements.

This is fair. We don't know what the best way to trigger reconnecting is yet. I still don't think pinging in both directions is wise, but I could be wrong.